nuxeo-core/nuxeo-core-mimetype/src/main/java/org/nuxeo/ecm/platform/mimetype/interfaces/MimetypeRegistry.java

```diff
@@ -169,4 +170,23 @@
      */
     String getMimetypeFromExtension(String extension);
 
+    /**
+     * Retrieves the normalized mime type for the give {@code mimeType}.
+     *
+     * @param mimeType the mime for which we are looking the normalized one
+     * @return an {@code Optional} with a present value if the normalised mime type is non-{@code null}, otherwise an
```

Comment:
normalised -> normalized
maybe "if the normalized mime type can be found" or "... exists"? (non-null is an implementation detail)